src/megachatapi.h

```diff
@@ -3109,6 +3108,19 @@ class MegaChatApi
      */
     MegaChatMessage *getMessage(MegaChatHandle chatid, MegaChatHandle msgid);
 
+    /**
+     * @brief Returns the MegaChatMessage specified from the chat room stored in node history
+     *
+     * This function allows to retrieve only those messages that are in the node history
+     *
+     * You take the ownership of the returned value.
+     *
+     * @param chatid MegaChatHandle that identifies the chat room
+     * @param msgid MegaChatHandle that identifies the message (msg id or a temporal id)
```

Comment:
A message in the node-history buffer is always confirmed. When they are in sending status they are not included in this buffer, only upon confirmation
```suggestion
     * @param msgid MegaChatHandle that identifies the message
```